tests/test_checker/test_noqa.py

```diff
@@ -100,6 +100,7 @@ def test_noqa_fixture_disabled(absolute_path, all_violations):
         'Z328': 2,
         'Z329': 1,
         'Z330': 1,
+        'Z331': 0,
```

Comment:
This set contains errors that must be present in `noqa.py`
Only exceptional rules might be missing. Please, provide at least one integration test